sdk/storage/azure-storage-blob/tests/test_blob_client.py

```diff
@@ -369,6 +368,60 @@ def test_create_service_with_connection_string_succeeds_if_secondary_with_primar
             self.assertTrue(service.primary_endpoint.startswith('https://www.mydomain.com/'))
             self.assertTrue(service.secondary_endpoint.startswith('https://www-sec.mydomain.com/'))
 
+    def test_create_service_with_custom_account_endpoint_path(self):
+        custom_account_url = "http://local-machine:11002/custom/account/path/" + self.sas_token
+        for service_type in SERVICES.items():
+            conn_string = 'DefaultEndpointsProtocol=http;AccountName={};AccountKey={};BlobEndpoint={};'.format(
+                self.account_name, self.account_key, custom_account_url)
+
+            # Act
+            service = service_type[0].from_connection_string(
+                conn_string, container_name="foo", blob_name="bar")
+
+            # Assert
+            self.assertEqual(service.account_name, self.account_name)
+            self.assertEqual(service.credential.account_name, self.account_name)
+            self.assertEqual(service.credential.account_key, self.account_key)
+            self.assertEqual(service.primary_hostname, 'local-machine:11002/custom/account/path')
+        
+        service = BlobServiceClient(account_url=custom_account_url)
+        self.assertEqual(service.account_name, "local-machine:11002")
```

Comment:
nit: do we want to put an account name there for customized url? Though it doesn't hurt because it seems we are not using it.